static/js/pages/committee-single.js

```diff
@@ -383,6 +408,11 @@ $(document).ready(function() {
           columns: disbursementPurposeColumns,
           callbacks: aggregateCallbacks,
           order: [[1, 'desc']],
+          hideEmptyOpts: {
+            dataType: 'disbursements',
+            name: context.name,
```

Comment:
Why is hound complaining so much about this here?  It looks like `context` is fine elsewhere, which has me wondering if there a variable definition or function argument missing.